core-services/prompt-registry/src/main/java/com/sap/ai/sdk/prompt/registry/OrchestrationConfigClient.java

```diff
@@ -0,0 +1,109 @@
+package com.sap.ai.sdk.prompt.registry;
+
+import static com.sap.ai.sdk.core.JacksonConfiguration.getDefaultObjectMapper;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.google.common.collect.Iterables;
+import com.sap.ai.sdk.core.AiCoreService;
+import com.sap.ai.sdk.prompt.registry.client.OrchestrationConfigsApi;
+import com.sap.ai.sdk.prompt.registry.model.AzureContentSafetyInputFilterConfig;
+import com.sap.ai.sdk.prompt.registry.model.AzureContentSafetyOutputFilterConfig;
+import com.sap.ai.sdk.prompt.registry.model.InputFilterConfig;
+import com.sap.ai.sdk.prompt.registry.model.LlamaGuard38bFilterConfig;
+import com.sap.ai.sdk.prompt.registry.model.OutputFilterConfig;
+import com.sap.cloud.sdk.cloudplatform.connectivity.ApacheHttpClient5Accessor;
+import com.sap.cloud.sdk.services.openapi.apiclient.ApiClient;
+import javax.annotation.Nonnull;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.springframework.http.client.BufferingClientHttpRequestFactory;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.http.converter.yaml.MappingJackson2YamlHttpMessageConverter;
+import org.springframework.web.client.RestTemplate;
+
+/**
+ * Client for managing Orchestration Configurations in the Prompt Registry service.
+ *
+ * @since 1.14.0
+ */
+public class OrchestrationConfigClient extends OrchestrationConfigsApi {
```

Comment:
(Major/Discussion)

I don't think we want to extend a custom class from generated code. Did we do that before? Is there no other way to apply mixins? Otherwise this has a code smell; we're introducing hand-written public API code for the sake of a workaround.